src/redux/reducers/app.js

```diff
@@ -18,13 +29,23 @@ export default handleActions(
       };
     },
     [actionTypes.APP_GET_LOGIN](state, { payload }) {
-      const { uuid, accessToken } = payload;
+      const { uuid, accessToken, requestPayload } = payload;
+      const email = requestPayload.login;
       return {
         ...state,
         uuid,
         accessToken,
+        email: accessToken ? email : '',
+      };
+    },
+    [actionTypes.APP_LOGOUT](state, { payload }) {
+      return {
+        ...defaultState,
       };
     },
   },
-  defaultState,
+  {
+    ...defaultState,
+    ...cookiesAppValues,
```

Comment:
Can't we use cookiesAppValues directly inside default state?

Reply:
No, because then the value of default state might look like
{
  uuid : '123433'
}

now when you do something like this,
{
 username: '',
...defaultState,
}

then you would always end up with '123433',
which would be wrong.

I could give you a more elaborate description of this situation.